Lib/glyphsLib/classes.py

```diff
@@ -373,6 +373,8 @@ def shouldWriteValueForKey(self, key):
 
 
 class Proxy:
+    __slots__ = "_owner"
```

Comment:
shouldn't `__slots__` be a tuple?

Reply:
It seems to work? :o

Reply:
funny. looks like it's a feature
https://github.com/python/typeshed/issues/3079#issuecomment-504157352